### ISSUE TITLE
Reset automatic assignment

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/nmrium#1488]
Before making an automatic assignment we should keep the original ranges / zones somewhere and be able to reapply them:

In the header of the automatic assignment panel we could add this icon for the reset:

![image](https://user-images.githubusercontent.com/1484241/164374577-42adf6cf-aef8-49b9-9ef0-967178d236c3.png)

**Comment by hamed-musallam** [comment]
@lpatiny 

I think we need to keep a copy of the spectra (ranges/zones/info) in the global state before we apply the auto assignments, What do you think?

**Comment by lpatiny** [comment]
I don't think it should be in the global state but should only be in the local state of the automatic assignment panel.
